src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,119 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exeptions.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+
+    private static final int ID_COLUMN = 1;
+    private static final int TITLE_COLUMN = 2;
+    private static final int PRICE_COLUMN = 3;
+    private static final int MIN_EXECUTE_UPDATE = 1;
+    private static final String ID = "id";
+    private static final String TITLE = "title";
+    private static final String PRICE = "price";
+
+    @Override
+    public Book create(Book book) {
+        String query = "INSERT INTO books (title,price) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
```

Comment:
Use Statement.RETURN_GENERATED_KEYS only in create statement, it's not needed in other methods.
improve your logic

Reply:
Not fixed